src/main/java/org/example/final02/controller/UserRegisterController.java

```diff
@@ -18,10 +22,24 @@ public UserRegisterController(MyUserRepository myUserRepository, PasswordEncoder
         this.passwordEncoder = passwordEncoder;
     }
 
-    @PostMapping("/register/user")
-    public MyUser registerUser(@RequestBody MyUser myUser) {
+    /*@PostMapping("/register/user")
+    public MyUser register(@RequestBody MyUser myUser) {
         myUser.setPassword(passwordEncoder.encode(myUser.getPassword()));
         return myUserRepository.save(myUser);
```

Comment:
whe it is commented?

Reply:
I tried a more suitable way

Reply:
normalne by zakomentovany kod nemal byt vobec v main pripadne stage